Clappr/Classes/Base/Core.swift

```diff
@@ -145,7 +145,12 @@ open class Core: UIBaseObject, UIGestureRecognizerDelegate {
     }
 
     open func setFullscreen(_ fullscreen: Bool) {
-        mediaControl?.fullscreen = fullscreen
+        guard let mediaControl = mediaControl, mediaControl.fullscreen != fullscreen else { return }
```

Comment:
Can we move this part `mediaControl.fullscreen != fullscreen` to a method and name it from what this concept represents?

Reply:
Reviewed 👍